tests/parsing/parchmint-parser.test.js

```diff
@@ -783,7 +903,134 @@ describe('terminals', () => {
     });
 });
 
+describe('schema validation', () => {
+    describe('valid', () => {
+        test('required keys (name, layers)', () => {
+            let pp = new ParchmintParser(validParchmintArchBareBones);
+            pp.parse();
+
+            expect(pp.valid).toBe(true);
+        });
```

Comment:
This feels like a test we've already done. Does this (or something close enough) cover this case?

Reply:
You're right this is the exact same test as the 'valid parse'. It's been removed.